main.py

```diff
@@ -16,10 +22,13 @@ def login():
     if 'name' not in data.keys() or 'password' not in data.keys():
         return 'Name or/and password are missing.', 400
     
-    if data['name'] == 'denchik' and data['password'] == 'foobar':
-        return 'Correct name and password.', 200
-    
-    return 'Wrong name or password.', 403
+    if data['name'] not in credit.keys():
+        return 'Such name is not registered.', 403
+
+    if not check_password_hash(credit[data['name']], data['password']):
```

Comment:
You should not differentiate between the fact that there is no such user and the fact that the password doesn't match as it exposes what users exist.

Reply:
done.